docs/querying.md

```diff
@@ -2,10 +2,83 @@
 
 The query macro and where clause combine to give you full control over your query.
 
+## Where
+
+Where is using a QueryBuilder that allows you to chain where clauses together to build up a complete query.
+```crystal
+posts = Post.where(published: true, author_id: User.first.id)
```

Comment:
For sake of working examples, probably should use `User.first!.id`, otherwise this would most likely throw the `no method id on (Int32 | Nil)` error